Book-A-Desk.Api/BookingNotifier.fs

```diff
@@ -90,21 +91,31 @@ module rec BookingNotifier =
             |> mailMessage.Cc.Add
         
         mailMessage.Subject <- emailNotificationDetails.Subject
-        let bodyPart = TextPart("plain")
+        let bodyPart = TextPart(TextFormat.Html)
         
         bodyPart.Text <- emailNotificationDetails.Text
         mailMessage.Body <- bodyPart
         mailMessage
-        
+
+    let healthQuestionnaireMailMessage =
+            $"SVP, veuillez répondre au questionnaire sur l’état de santé. Si vous répondez oui à une des questions, nous vous demandons de rester à la maison.{Environment.NewLine}{Environment.NewLine}" +
+            $"Si vous avez d’autres questions, n’hésitez pas à écrire à <a href=\"mailto:Anastasia.vlahos@broadsign.com\">Anastasia.vlahos@broadsign.com</a>.{Environment.NewLine}{Environment.NewLine}" +
+            $"We ask that you please fill out our health questionnaire. Should you answer yes to any of the questions, we politely ask that you refrain from coming into the office.{Environment.NewLine}{Environment.NewLine}" +
+            $"If you have any questions at all, please feel free to reach out to <a href=\"mailto:Anastasia.vlahos@broadsign.com\">Anastasia.vlahos@broadsign.com</a>.{Environment.NewLine}{Environment.NewLine}" +
+            $"Link:  <a href=\"https://forms.gle/FyzgXfst8FYRukih9\">Questionnaire sur la Santé // Health Questionnaire</a>.{Environment.NewLine}"
+
     let createMailText (bookingDate : DateTime) (office : Office) =
             let (CityName officeName) = office.City
             $"You have booked a desk at %s{bookingDate.ToShortDateString()} in the Office %s{officeName}{Environment.NewLine}" +
             $"Opening hours: {office.OpeningHoursText}{Environment.NewLine}" +
-            "It is your responsibility to verify that the office is open for the date that you have booked"
-    
+            $"It is your responsibility to verify that the office is open for the date that you have booked.{Environment.NewLine}{Environment.NewLine}" +
+            healthQuestionnaireMailMessage
+
     let createOfficeRestriction (office : Office) =
             let (CityName officeName) = office.City
-            "We inform you that the office has some restrictions."
+            $"We inform you that the Office %s{officeName} has some restrictions.{Environment.NewLine}" +
```

Comment:
"the **O**ffice" => the office (to keep consistency with "the office" in the line above)

Reply:
I copied "the Office" from message sent in the confirmation. Do we need to remove uppercase in both? What do you think?